content/cpp/concepts/queues/terms/empty/empty.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '.empty()'
+Description: 'Checks if queue has no elements.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.empty` method returns `true` if the [size](https://www.codecademy.com/resources/docs/cpp/queues/size) of the queue is 0. Otherwise, it returns `false`.
```

Comment:
```suggestion
The `.empty()` method returns `true` if the [size](https://www.codecademy.com/resources/docs/cpp/queues/size) of the queue is 0. Otherwise, it returns `false`.
```